ipfx/data_set_features.py

```diff
@@ -207,6 +218,13 @@ def extract_cell_features(data_set,
     return cell_features
 
 
+def stim_align_sweep_set(sweep_set):
+
+    for sweep in sweep_set.sweeps:
+        expt_start_idx, _ = ep.get_experiment_epoch(sweep.i,sweep.sampling_rate)
```

Comment:
if you detect the experiment epoch, you could have this function take the epoch name as an argument, as opposed to detecting a specific epoch here.